src/app_service/service/community/dto/community.nim

```diff
@@ -141,9 +141,10 @@ proc toCommunityDto*(jsonObj: JsonNode): CommunityDto =
     result.permissions = toPermission(permissionObj)
 
   var membersObj: JsonNode
-  if(jsonObj.getProp("members", membersObj)):
-    for memberId, memberObj in membersObj:
-      result.members.add(toMember(memberObj, memberId))
+  if (jsonObj["members"].kind != JNull):
```

Comment:
@jrainville for this block here we're still not guarded enough if we just check if members object is not null, cause we may have a crash if members object is not null, but it's any other but the `JArray` kind (it's always better to filter out exactly what you expect). So if you agree I would rather do something like this:
```
var membersObj: JsonNode
if(jsonObj.getProp("members", membersObj) and membersObj.kind == JArray):
    for memberId, memberObj in membersObj:
        result.members.add(toMember(memberObj, memberId))
```